README.md

```diff
@@ -505,8 +505,8 @@ the COOL environment.
 | debian\_desktop\_security\_group | The security group for the Debian desktop instances. |
 | efs\_client\_security\_group | A security group that should be applied to all instances that will mount the EFS file share. |
 | efs\_mount\_targets | The mount targets for the EFS file share. |
-| email\_sending\_domain\_certreadrole | The IAM role that allows for reading the certificate for the email-sending domain. |
-| gophish\_instance\_profile | The instance profile for the Gophish instances. |
+| email\_sending\_domain\_certreadrole | The IAM roles that allow for reading the certificate for the email-sending domain. |
+| gophish\_instance\_profile | The instance profiles for the Gophish instances. |
```

Comment:
Why do these outputs have singular names?  I think plural names (e.g. `email_sending_domain_certreadroles`) would make more sense, and would be more consistent with the input variable `email_sending_domains` above.

Reply:
Agreed - please see https://github.com/cisagov/cool-assessment-terraform/pull/153/commits/80a13efd98e839e727fc46b81bc8a35fb4d421fc.